tests/security/atsec/setup_atsec_env.pm

```diff
@@ -19,7 +19,7 @@ sub run {
 
     select_console 'root-console';
 
-    zypper_call('in wget gcc make');
+    zypper_call('in wget gcc make curl');
```

Comment:
you need to squash all your commit (I saw 2 debug ones I think) all in this PR seems related with the UI, except this one change that I would recommend to put it in a different commit, make sense? (just in case something need to be reverted, we hope not! :) you don't need to revert stuff that is fine).

Reply:
we can keep it together. This is actually needed because of this new installation method being used. With autoyast it wasn't required since curl was already installed.